README.md

```diff
@@ -63,7 +63,7 @@ Web version is also available now. but only transmitter version. [Web version.]
 
 
 ## Contributing
-Changes are improvements are more than welcome! Feel Free to fork and open a pull request. Please make your changes in a specific branch and request to pull into `master`.
+Changes and improvements are more than welcome! Feel Free to fork and open a pull request. Please make your changes in a specific branch and request to pull into `master`.
```

Comment:
This modification is already applied in euphony:master.
So, I suggest you to remove commits related to ```README.md``` file.
And, co-author is not applied at 75448bd. Try using git commit amend and adding co-author.
Or, reset hard to master, create new commit and force update.

Please edit this PR so that there is only one commit with GETTING_STARTED.md and GETTING_STARTED_KR.md added :)

Reply:
I agree with you :)

Reply:
yes, i will fix that!